empty/public/phaser/main.js

```diff
@@ -1,19 +1,23 @@
 //create main state that contains game
 
-const mainState = {
+let id
+
+var mainState = {
   preload: function() {
     // Load the sprites
-    game.load.image('nyan', 'assets/NyanCat.png')
-    game.load.image('bird', 'assets/bird.png')
-    game.load.image('bullet', 'assets/bullet.png')
-    game.load.image('pipe', 'assets/pipe.png')
-
-    //load jump sound
-    game.load.audio('jump', 'assets/jump.wav')
+    game.load.image('nyan', 'phaser/assets/NyanCat.png')
```

Comment:
There's a lot going on in this file.  Consider separating individual chunks of logic out - `preload` could be a function imported from another module, for example, so that the responsibility of this model is just to create the initial main state by importing and aggregating all the smaller pieces.

This particular function could also be cleaned up a little (and made a little easier to update):
```js
const ASSETS = [ 
  { type: 'image', name: 'nyan', extension: 'png' }, 
  { type: 'image', name: 'bird', extension: 'png' }, 
  /* etc. */
  { type: 'audio', name: 'jump', extension: 'wav' } 
]

const preload = () => ASSETS.forEach( { type, name, extension } => 
  game.load( type, `phaser/assets/${name}.${extension}` )
)
```